process/elasticproc/logsevents/nftsProcessor.go

```diff
@@ -159,6 +159,7 @@ func (np *nftsProcessor) convertMetaData(metaData *esdt.MetaData) *alteredAccoun
 		return nil
 	}
 
+	encodedCreatorAddr := np.pubKeyConverter.SilentEncode(metaData.Creator, log)
```

Comment:
Are we sure we want silent enconding here?

Reply:
added a log.Warn in case of error